### PR TITLE
Change aio links to adev

### PR DESCRIPTION
**Pull request by vimulatus**
Refer #780

### CODE REVIEW

File: docs/src/content/docs/challenges/angular/16-di.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
 title: 🔴 Master Dependancy Injection
```
Comment (svenson95) [inline docs/src/content/docs/challenges/angular/16-di.md:2]: here is a typo as well -> Dependancy should be Dependency

**Comment by tomalaforge** [comment]
You need to rebase your branch, so I can review it.

**Comment by vimulatus** [comment]
@tomalaforge is this ok now ?

**Review by tomalaforge — APPROVED** [review]

**Comment by tomalaforge** [comment]
Thanks a for doing it. Great work

**Pull request merged**